corehq/apps/smsbillables/models.py

```diff
@@ -367,3 +399,51 @@ def _get_usage_fee(cls, domain, direction):
                 % (direction, domain)
             )
         return usage_fee
+
+    @classmethod
+    def _get_twilio_charges(cls, backend_message_id, direction, couch_id):
+        def _get_twilio_client():
+            twilio_backend = SQLTwilioBackend.get_global_backends_for_this_class(SQLTwilioBackend.SMS).get()
```

Comment:
We should never get here with domain-owned backends though, due to https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/smsbillables/models.py#L333

Although if we added a second twilio global backend this code might need updating.